3.0-rc/alpine3.11/Dockerfile

```diff
@@ -0,0 +1,130 @@
+FROM alpine:3.11
```

Comment:
This file should be dropped completely -- we don't support newer releases on older base operating systems unless there's a really compelling reason to do so (and for Alpine 3.11 vs 3.12, there isn't a strong one we've seen).